_posts/0000-01-02-takahiroinaba1113.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
Replace this line with this message
And meet the suggestion of using multiple lines